### ISSUE TITLE
MS Edge: NVDA cannot navigate letter by letter in a group of elements when pressing left and right arrow

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8306]
### Steps to reproduce:
1. Open this website in MS Edge
https://groups.google.com/forum/#!forum/braillists
2. In NVDA's settings, go to Browse mode and disable "Use screen layout, if available"
3. On google groups, press b to go to "new"
4. Press left and right arrow to read the letters of the element

### Expected behavior:
NVDA should read the letters like in any other browser.

### Actual behavior:
NVDA jumps between buttons when pressing left and right arrow key.

### System configuration:
NVDA version:
NVDA 2018.1.1

NVDA Installed or portable:
Installed

Windows version:
Windows 10.0.17134

Name and version of other software in use when reproducing the issue:
MS Edge

### Other questions:

Does the issue still occur after restarting your PC?
Yes.

Have you tried any other versions of NVDA?
No

**Comment by leonardder** [comment]
Note that the screen layout setting has no effect in Edge as far as I recall. If I understand the issue correctly, the buttons have no visible label, and therefore they are treated as only one character.
@michaeldcurran: Is there something to do about this?

**Comment by Adriani90** [comment]
Closing as works for me, in Edge Chromium 81 everything works fine.

**Comment by Adriani90** [comment]
Adding appropriate label since this is definitely still reproducible in Edge Spartan and UIA related environments.
